.gitignore

```diff
@@ -22,6 +22,7 @@ platform/solr/**/overlays
 distribution/docs/src/main/resources/content/_catalogFrameworks/.asciidoctor
 distribution/docs/src/main/resources/content/_catalogFrameworks/catalog_architecture_framework.png
 distribution/docs/src/main/resources/content/_catalogFrameworks/ingest_request.png	
+distribution/docs/src/main/resources/content/_catalogFrameworks/ingest_request.png
```

Comment:
✏️  I might be missing it but lines 24 and 25 look the same.

Reply:
fixed